index.html

```diff
@@ -26,15 +26,133 @@ <h2 class="introh2">LET’S CONNECT</h2>
 
       <div class="socials">
         <ul>
-          <li class="ls"><a href="#" class="twitter-hp"><img src="/images/twitter.svg" alt="This is a twitter button"></a></li>
-          <li class="ls"><a href="#" class="linkedin-hp"><img src="/images/Linkedin icon.svg" alt="Thi is a linkedin button"></a></li>
-          <li class="ls"><a href="#" class="unknw1"><img src="/images/Vector.svg" alt=""></a></li>
-          <li class="ls"><a href="#" class="github-hp"><img src="/images/github.svg" alt="This is a github button"></a></li>
-          <li class="ls"><a href="#" class="unknw2"><img src="/images/Vector (1).svg" alt=""></a></li>
+          <li class="ls"><a href="#" class="twitter-hp"><img src="./images/twitter.svg" alt="This is a twitter button"></a></li>
+          <li class="ls"><a href="#" class="linkedin-hp"><img src="./images/Linkedin icon.svg" alt="Thi is a linkedin button"></a></li>
+          <li class="ls"><a href="#" class="unknw1"><img src="./images/Vector.svg" alt=""></a></li>
+          <li class="ls"><a href="#" class="github-hp"><img src="./images/github.svg" alt="This is a github button"></a></li>
+          <li class="ls"><a href="#" class="unknw2"><img src="./images/Vector (1).svg" alt=""></a></li>
         </ul>
       </div>
     </div>
   </section>
-</body>
 
+<!-- WORKS/PROJECTS-->
+
+<section class="works1">
+  <section class="card1">
+    <img class="card1-img" src="./images/Snapshoot Portfolio.png" alt="This is an image for work one">
+      <h1 class="card1-h">Tonic</h1>
+          <ul class="card1-ul1">
+              <li class="card1-canopy">CANOPY</li>
+              <li class="card1-back-end-dev">Back End Dev</li>
+              <li class="card1-2015">2015</li>
+          </ul>
+      <p class="card1-paragraph">A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups required.</p>
+      <ul class="card1-ul2">
+          <li class="card1-html">HTML</li>
+          <li class="card1-css">CSS</li>
+          <li class="card1-javascript">JavaScript</li>
+      </ul>
+      <button class="card1-button">See project</button>
+    </section>
+</section>
+
+<section class="works2">
+  <section class="card2">
+    <img class="card2-img" src="./images/port2.png" alt="This is an image for work one">
+      <h1 class="card2-h">Multi-Post <br> Stories</h1>
+          <ul class="card2-ul1">
+              <li class="card2-canopy">CANOPY</li>
+              <li class="card2-back-end-dev">Back End Dev</li>
+              <li class="card2-2015">2015</li>
+          </ul>
+      <p class="card2-paragraph">A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups required.</p>
+      <ul class="card2-ul2">
+          <li class="card2-html">HTML</li>
+          <li class="card2-css">CSS</li>
+          <li class="card2-javascript">JavaScript</li>
```

Comment:
For this section, I recommend the following:

- To make the code well structured, consider not to wrap a `section` tag within another since it can cause some problems with the accessibility of your website and can also make it hard to follow by other devs
- The heading tag for the project card should not be an h1 tag since this tag is mostly used to open up a website mostly at the hero section, hence I recommend using maybe an h3 here instead of h1 to follow best practices.
- For interactivity purposes, the button here and for other occurrences of a button throughout the website should have a hover and pressed/active states.